lib/platform/subscriptions.js

```diff
@@ -100,12 +100,11 @@ module.exports = class Subscriptions extends Base {
 	}
 
 	// Why does this accept an array of device IDs but devices does not
-	subscribeToDeviceHealth(devices, subscriptionName) {
+	subscribeToDeviceHealth(subscriptionName) {
 		const path = `installedapps/${this.st.installedAppId}/subscriptions`
 		const body = {
 			sourceType: 'DEVICE_HEALTH',
 			deviceHealth: {
-				deviceIds: devices.map(it => it.deviceConfig.deviceId),
```

Comment:
In the description of the source type itself it says _"Details of a subscription of source type DEVICE_HEALTH. Only one of deviceIds or locationId should be supplied."_

I elected to not support both options because there can only be one device health handler